atom/browser/native_window_mac.mm

```diff
@@ -382,6 +393,20 @@ - (void)drawRect:(NSRect)dirtyRect {
   // We will manage window's lifetime ourselves.
   [window_ setReleasedWhenClosed:NO];
 
+  // Configure title bar look on Yosemite or newer
+  if (base::mac::IsOSYosemiteOrLater()) {
+    if ((titleBarStyle == "hidden") || (titleBarStyle == "hidden-inset")) {
+      [window_ setTitlebarAppearsTransparent:YES];
```

Comment:
Can you create a new issue for this? I'm wondering if it is caused by me setting `MACOSX_DEPLOYMENT_TARGET` to `10.10`.